u-root.go

```diff
@@ -304,7 +304,6 @@ func Main() error {
 				Packages: pkgs,
 			},
 		},
-		Archiver:        archiver,
```

Comment:
Why can't we keep the archiver? We were planning on adding "iso" and "docker" in the future.